torch/_subclasses/fake_tensor.py

```diff
@@ -1437,12 +1756,6 @@ def dispatch(self, func, types, args=(), kwargs=None):
         elif func is torch.ops.aten.storage_offset.default:
             return int(args[0].storage_offset())
 
-        if log.getEffectiveLevel() <= logging.DEBUG:
```

Comment:
Looks like I moved this below which I don't think was necessary. I'll move it back.